### DISCUSSION TITLE
Making my own interstellar unblocked link

### DISCUSSION THREAD

**Discussion by adamsabbaghiscool** [UseInterstellar/Interstellar#908] [Q&A]
I've been using interstellar unblocked links for a while now but I've always trying to know how to make one myself so can you help me make one ?

**Comment by lakshinhemachandran** [comment]
I've also been trying to do this, but the code is way too complicated
You have to know at least three coding languages to even start on that type of project.

**Reply by Aymano200** [reply]
Not even you can just deploy it on vercel with the guide on docs of interstellar

**Reply by Shalomisthegoatandyouarenot** [reply]
Sigma Sigma

**Reply by cute878** [reply]
idk how to get an unblocker

**Reply by MonkeyPrince813** [reply]
Go to unblock github.com

**Comment by Charlieis1nsan3** [comment]
blud you guys are retarded

**Comment by Charlieis1nsan3** [comment]
Visit FreeDNS Go to FreeDNS.  (you can't use an unblocker for this site, it won't work). Sign Up for a Free Account If this is your first time, click "Sign Up for Free." If you already have an account, proceed to Step 5. Fill Out Your Details Complete the sign-up form with your details: Name: You don't have to use your real name. User ID: Choose a unique ID. Email: You can use a temporary email provider like TempMail to create more links in the future. Complete Email Activation Once you've filled in the details, click "Sign Up." You’ll receive an activation email. Confirm your email address to complete the registration. Register Your Link After creating your account, head to the FreeDNS registration page: FreeDNS Registry. Alternatively, on the FreeDNS dashboard: Click on "Subdomains" in the left-hand menu. Select "Add Subdomain." In the "Domain" dropdown, choose "many, many more." Click on the first sentence that says "Shared Domain Registry." Choose a Domain Browse the list of domains available for use (e.g., chickenkiller.com). Ensure that the domain you choose is accessible to you and not blocked. Select a domain that has low popularity for higher chances of being unblocked. Make sure the domain is public. Configure the Domain On the configuration page: Make sure the "A" record type is selected (this should be the default). Enter the name of your subdomain (e.g., subdomainName.chosenDomain.com). Set the Destination Depending on which unblocker you are using, enter the correct IP destination for your subdomain. Save Your Settings After entering the IP destination, click "Save" and complete the captcha.

**Reply by Charlieis1nsan3** [reply]
the IP for interstellar is 5.161.66.1 fucking monkeys

**Comment by cute878** [comment]
hey

**Comment by Grayso5** [comment]
So i need help making a proxy ive tried sevral times and just cant figure it out i know html css and javascript can someone please help here is my website currently https://iavcwk.mimo.run/index.html

**Comment by cleverman606** [comment]
If you want to make an unblocked link. You must follow this: Purchase a domain (free ones don't work because 99% of the time they are blocked). Set up the primary domain (example.com) to point to something that looks educational: mydomain.dev (my case). Then, wait a little bit. Visit your site on your laptop to make sure your school block company gets your message, and then wait a little bit around a week. Your site will be unblocked. you can now create a subdomain to point to Interstellar or anything, and you will now have your unblocked site. This works with many companies including Lightspeed (mine schools block). got passed it like this
